package.json

```diff
@@ -5,24 +5,25 @@
   "author": "Chainsafe Systems",
   "license": "MIT",
   "scripts": {
-    "start": "craco start",
-    "build": "craco build",
+    "start": "GENERATE_SOURCEMAP=false craco start",
```

Comment:
see [here](https://github.com/facebook/create-react-app/issues/11924#issuecomment-1016947373) for rationale.